go-controller/pkg/cni/helper_linux.go

```diff
@@ -252,6 +253,13 @@ func setupInterface(netns ns.NetNS, containerID, ifName string, ifInfo *PodInter
 			ifnameSuffix = fmt.Sprintf("_%d", containerVeth.Index)
 		}
 
+		// If we have the ipv6 gateway LLA then this is a primary layer2 UDN
+		if len(ifInfo.GatewayIPv6LLA) > 0 {
+			if err = setupIngressFilter(ifInfo.GatewayIPv6LLA.String()); err != nil {
+				return err
```

Comment:
maybe wrap this error ? indicating we could not install the RA filtering rules ?